scripts/tvm_cli/tvm_cli.py

```diff
@@ -157,6 +158,28 @@ def compilation_preprocess(args):
         info['namespace'] = path.sep.join(namespaces[-4:])
         info['network_name'] = namespaces[-2]
 
+    # Get version information from the most recent tag.
+    # Default to 0.0.0 when the git tag information can't be accessed.
+    info['modelzoo_version'] = [0, 0, 0]
+    modelzoo_rootdir = path.abspath(path.dirname(__file__) + "/../../")
+    if path.isdir(path.join(modelzoo_rootdir, '.git')):
+        cmd = 'git describe --tags --match [0-9]*'.split()
+        try:
+            version = subprocess.check_output(cmd).decode().strip().split('-', maxsplit=1)[0]
+            version = version.split('.')
+            if len(version) == 3:
+                info['modelzoo_version'] = version
+            else:
+                print(
+                    f"Unexpected git tag, descriptor version defaults to "
+                    f"{info['modelzoo_version']}"
+                )
+        except subprocess.CalledProcessError:
+            print(
+                f"No git tag information, descriptor version defaults to "
+                f"{info['modelzoo_version']}"
+            )
```

Comment:
If exception occurs, it seems that package would be overwritten with `0.0.0` and named like `centerpoint_backbone-x86_64-llvm-0.0.0-20221003.tar.gz` for example?

Reply:
That will only have an impact on the value that will be written in the `inference_engine_tvm_config.hpp` helper file, not the archive name. The version number in the `.tar.gz` file produced in this repo's CI comes from the tag pushed, so it will not be 0.

But if a user compiles a model locally and an exception occurs, the version defaults to `0.0.0`. In the draft PR in Autoware that aligns with this change (https://github.com/autowarefoundation/autoware.universe/pull/2158) I am mapping a 0 version value in the header file to "Unknown", and it shouldn't have much impact except potentially a warning, but that would be up to the actual packages that make use of the neural network.